content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -396,49 +372,16 @@ to restrict access to only the appropriate callers.
     caption="A service mesh with two namespaces and an exposed service"
     >}}
 
-When using [multiple clusters](#multiple-clusters), the namespaces in each
-cluster sharing the same name are considered the same namespace. For example,
-`Service B` in the `foo` namespace of `cluster-1` and `Service B` in the
-`foo` namespace of `cluster-2` refer to the same service, and Istio merges their
-endpoints for service discovery and load balancing.
+### Cluster Tenancy
 
-{{< image width="50%"
-    link="cluster-ns.svg"
-    alt="A service mesh with two clusters with the same namespace"
-    title="Multicluster namespaces"
-    caption="A service mesh with clusters with the same namespace"
-    >}}
-
-### Cluster tenancy
-
-Istio supports using clusters as a unit of tenancy. In this case, you can give
-each team a dedicated cluster or set of clusters to deploy their
-workloads. Permissions for a cluster are usually limited to the members of the
-team that owns it. You can set various roles for finer grained control, for
-example:
-
-- Cluster administrator
-- Developer
-
-To use cluster tenancy with Istio, you configure each cluster as an independent
-mesh. Alternatively, you can use Istio to implement a group of clusters as a
-single tenant. Then, each team can own one or more clusters, but you configure
-all their clusters as a single mesh. To connect the meshes of the various teams
-together, you can federate the meshes into a multi-mesh deployment.
-
-{{< image width="50%"
-    link="cluster-iso.svg"
-    alt="Two isolated service meshes with two clusters and two namespaces"
-    title="Cluster isolation"
-    caption="Two isolated service meshes with two clusters and two namespaces"
-    >}}
+You can use the cluster as the unit of isolation in a mesh by giving each team
+a dedicated cluster, with its [own control plane](#independent-control-plane) allowing
+them to manage their own configurations. Alternatively, if you can ensure that namespaces
+are globally unique across clusters, you can use a shared control plane.
 
-Since a different team or organization operates each mesh, service naming
-is rarely distinct. For example, the `mysvc` in the `foo` namespace of
-`cluster-1` and the `mysvc` service in the `foo` namespace of
-`cluster-2` do not refer to the same service. The most common example is the
-scenario in Kubernetes where many teams deploy their workloads to the `default`
-namespace.
+### Mesh Tenancy
```

Comment:
So it appears that you removed the `Mesh models` section in favor of using mesh as a unit of tenancy. I'm not entirely against this as it does seem to simplify this a bit.

@smawson thoughts?